src/main.c

```diff
@@ -1,25 +1,23 @@
 #include "cli.h"
 #include "file.h"
 #include "store.h"
+#include "linda.h"
 #include "debug.h"
 
 #define FILENAME "linda.file"
```

Comment:
W specyfikacji jest mowa o plikach konfiguracyjnych. My jak rozumiem ich nie będziemy używać?

Reply:
Jakie pliki konfiguracyjne?

Reply:
Co powinien zawierać projekt ostateczny:
![image](https://user-images.githubusercontent.com/17070569/34791520-84e4bf0c-f645-11e7-807f-42281a6b7d17.png)
Wydaje mi się że nie są konieczne ale poruszam kwestię

Reply:
Jak będzie czas to będą pliki konfiguracyjne